modules/@apostrophecms/schema/ui/apos/components/AposInputString.vue

```diff
@@ -67,6 +79,11 @@ export default {
       }
     }
   },
+  watch: {
```

Comment:
If the `type` property is not going to change while the component is in place, I don't think we need to watch it. `defineStep` could run on `mounted` and look at the `field.type` directly.

Reply:
Oh, I see you are doing that already. Are we expected it would also change? It doesn't seem like we need to watch this.

Reply:
You are right. No need to watch.